packaging/windows/generatemsi.ps1

```diff
@@ -144,18 +213,35 @@ if(-Not (RunCandle))
     Exit -1
 }
 
+if(-Not (RunCandleForBundle))
+{
+    Exit -1
+}
+
 if(-Not (RunLight))
 {
     Exit -1
 }
 
+if(-Not (RunLightForBundle))
+{
+    Exit -1
+}
+
 if(!(Test-Path $DotnetMSIOutput))
 {
     throw "Unable to create the dotnet msi."
     Exit -1
 }
 
+if(!(Test-Path $DotnetBundleOutput))
+{
+    throw "Unable to create the dotnet bundle."
+    Exit -1
+}
+
 Write-Host -ForegroundColor Green "Successfully created dotnet MSI - $DotnetMSIOutput"
```

Comment:
What about the download experience? Does the user download both the exe and msi to install?